llama_index/indices/multi_modal/base.py

```diff
@@ -62,7 +64,13 @@ def __init__(
         storage_context = storage_context or StorageContext.from_defaults()
 
         if image_vector_store is not None:
-            storage_context.add_vector_store(image_vector_store, self.image_namespace)
+            if self.image_namespace not in storage_context.vector_stores:
+                storage_context.add_vector_store(
```

Comment:
Should we just always override the existing image store if it's passed in as a kwarg?

Reply:
since the `if` condition will always be `False` I think

Reply:
ah wait, nvm, I see -- the storage context only adds the namespace if explicitly passed in